<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Device_Select</class>
 <widget class="QWidget" name="Device_Select">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">DeviceSelect</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>请选择一个设备定义文件（devices.json）</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="currentDirButton">
       <property name="text">
        <string>当前目录</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="FileSelector" name="fileSelector" native="true"/>
   </item>
   <item>
    <widget class="QWidget" name="actionsWidget" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="labelAlignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>创建新设备</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="newDeviceButton">
        <property name="text">
         <string>配置…</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>编辑现有设备</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="existingDevicesComboBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="deviceIndicatorLabel">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileSelector</class>
   <extends>QWidget</extends>
   <header>ui.implements.components.fileSelector</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
